site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -185,14 +185,12 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`              | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition | `false`                |
 | `helm.installTests`                                 | Add an ability to run `helm test agones` to verify the installation                             | `8000`                 |
 
-{{% feature publishVersion="1.17.0" %}}
```

Comment:
For this one, just bump the version number to 1.18.0 and leave the table in place.

Reply:
Done.